otter/test/test_auth.py

```diff
@@ -743,8 +746,8 @@ def test_authenticate(self):
         mock_auth = iMock(IAuthenticator)
         mock_auth.authenticate_tenant.return_value = succeed(result)
         log = object()
-        intent = Authenticate(mock_auth, 'tenant_id1', log)
-        self.assertEqual(self.successResultOf(intent.perform_effect(None)), result)
+        eff = Effect(Authenticate(mock_auth, 'tenant_id1', log))
+        self.assertEqual(sync_perform(get_dispatcher(None), eff), result)
```

Comment:
Non-blocking FYI: Its a little strange to use `sync_perform` when `Authenticate` is a `deferred_performer`.